cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
is this file doing something? 🤔

Reply:
I don't know. It's generated by cypress installation

Reply:
can you try deleting it?

Reply:
these files are important and you will likely need them soon to customize the way that you use cypress and create generic commands to have less repetitive code in your tests